spec/lib/govuk_component/tag_helpers_spec.rb

```diff
@@ -34,4 +34,31 @@
       end
     end
   end
+
+  describe '#govuk_tag_active_user?' do
+    it 'responds to govuk_tag_active_user?' do
+      expect(helper).to respond_to(:govuk_tag_active_user?)
+    end
+
+    context 'when external user is inactive' do
+      let(:user) { create(:user, :active, :disabled) }
+
+      it 'displays a red inactive tag' do
+        tag_class = 'govuk-tag govuk-tag--red'
+        tag_text = 'Inactive'
+
+        expect(govuk_tag_active_user?(user)).to have_tag(:strong, with: { class: tag_class }, text: tag_text)
+      end
```

Comment:
In helper specs `helper` is made available so that you can call methods on it, such as:

```ruby
helper.govuk_tag_active_user?(user)
```

This will mean that you do not need to `include SharedHelpers` in the helper module.

More broadly, it is good to separate out the test set up from the execution, so you can put `tag_class` and `tag_text` in `let`s and `helper.govuk_tag_active_user?(user)` in a `subject`. Then you can have;

```suggestion
      subject { helper.govuk_tag_active_user?(user) } # Further up, outside the contexts

      let(:tag_class) { 'govuk-tag govuk-tag--red' }
      let(:tag_text) { 'Inactive' }

      it { is_expected.to have_tag(:strong, with: { class: tag_class }, text: tag_text) }
```

Reply:
Thanks @jrmhaig I didn't realise that about the helper - thank you. That makes sense.